geopandas/io/tests/test_io.py

```diff
@@ -1,12 +1,33 @@
 from __future__ import absolute_import
+import sqlite3
+import os
 
 import fiona
+import pytest
 
 import geopandas
-from geopandas import read_postgis, read_file
+from geopandas import read_postgis, read_file, read_sqlite
+from geopandas.tests.util import (connect, create_postgis, create_sqlite,
+                                  validate_boro_df)
+
+
+@pytest.fixture
+def nybb_df():
+    nybb_path = geopandas.datasets.get_path('nybb')
+    df = read_file(nybb_path)
+
+    return df
 
-import pytest
-from geopandas.tests.util import connect, create_db, validate_boro_df
+
+def test_read_sqlite(tmpdir, nybb_df):
```

Comment:
Per recent issue and PR from @cr458, anticipating a little further cleanup in this module with pytest fixtures, but decided to deter that a future PR, since the scope of this PR has already broadened somewhat beyond the original SRID intent. :smile: